include/logger.h

```diff
@@ -0,0 +1,28 @@
+#ifndef LOGGER_H
+#define LOGGER_H
+
+class Vector3f;
+class Vector4f;
+class Matrix3f;
+class Matrix4f;
+class Spectrum;
+class HitRecord;
+class Ray;
+class Triangle;
+class MeshTriangle;
+class MeshData;
+
+class Logger {
```

Comment:
the final idea is to make this a singleton class. Currently, this is stateless. Later, this class could however store the path to its log file (or even an output stream).

Reply:
I like it! Having logging code in one place makes sense to me.